test/TestProviderRound.spec.js

```diff
@@ -5,40 +5,56 @@ require('truffle-test-utils').init();
 contract('ProviderRound', accounts => {
 
   let providerRound;
+  let contractAddress;
+  const PROVIDER_POOL_SIZE = 5;
   const PROVIDER_NULL = 0;
   const PROVIDER_REGISTERED = 1;
 
+  async function approveBondProvider(pricePerStorageMineral, pricePerComputeMineral, blockRewardCut, feeShare, amountBonded, providerAddress) {
+      await providerRound.approve(contractAddress, amountBonded, {from: providerAddress});
```

Comment:
I'm actually unable to find this function in `ProviderRound.sol` - what is this doing?

https://github.com/transmute-industries/network-contracts/blob/2901e6c3fbe9cd4f574bb0629c487560b01b9892/contracts/ProviderRound.sol

Reply:
Ah, this is a function of the `MintableToken`. 

Can we get a little detail on that here with a note to describe exactly what this is doing - `approve` is pretty generic.